cmd/snap-exec/main.go

```diff
@@ -114,7 +114,7 @@ func snapExec(snapApp, revision, command string, args []string) error {
 
 	cmd, err := findCommand(app, command)
 	if err != nil {
-		return err
+		return fmt.Errorf("cannot find command %q in app %q: %s", command, app, err)
```

Comment:
actually we control the findCommand errors in the same file,  so return err would be ok

we might tweak them though if we want